Rakefile

```diff
@@ -0,0 +1,4 @@
+task :run_crawler do
+  puts "rake task running"
+  `ruby /Users/shane/work/Morrigan/script.rb`
```

Comment:
Can you make this path in some way relative. As it is, this will only work on your computer.

Reply:
Probably going to delete this rake task all together, as it's not adding a ton of value